avalon/lib.py

```diff
@@ -281,3 +282,23 @@ def modules_from_path(path):
         modules.append(module)
 
     return modules
+
+
+def find_submodule(module, submodule):
+    """Find and return submodule of the module.
+
+    Args:
+        module (types.ModuleType): The module to search in.
+        submodule (str): The submodule name to find.
+
+    Returns:
+        types.ModuleType or None: The module, if found.
+
+    """
+    name = "%s.%s" % (module.__name__, submodule)
```

Comment:
Maybe also use `.format()` here to be consistent?

Reply:
Resolved with: https://github.com/getavalon/core/pull/501/commits/5aea369f9d3818d89c81297a1e32764d34180c5f